lib/vanagon/platform/windows.rb

```diff
@@ -82,6 +85,8 @@ def generate_msi_package(project)
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
         "mkdir -p $(tempdir)/#{project.name}/staging",
+        "mkdir -p $(tempdir)/wix",
+        "#{@copy} *.wxs $(tempdir)/wix",
```

Comment:
@jpogran and @melissa Just need to be careful on conflicts on these lines as all 3 of us have made slightly different changes (e.g. Melissa has dropped the project.name from the staging tree and I think we might a cp -r to copy the UI sub-directories (with icons/gifs)) over.

However - we can probably deal with this once we get the first set of PR's merged.